scala_core/src/main/scala/org/nimbleedge/envisedge/Aggregator.scala

```diff
@@ -12,23 +12,36 @@ import akka.actor.typed.scaladsl.AbstractBehavior
 import akka.actor.typed.Signal
 import akka.actor.typed.PostStop
 
+  /**
+      *The FL system is comprised of three major entities,
+      *and the aggregator serves as an intermediary between
+      *the orchestrator and the trainer. The aggregator can
+      *have children, such as other aggregators and trainers.
+      *What we are creating here is an aggregator object.
```

Comment:
You can re-write it like, Using the 
```suggestion
      *The FL system is comprised of three major entities,
      *and the aggregator serves as an intermediary between
      *the orchestrator and the trainer. The object of aggregator can
      *have children, such as other aggregators and trainers.
```